deployment/keystone/changeset/helpers_test.go

```diff
@@ -0,0 +1,353 @@
+	// set the env addresses to the deployed addresses that were created prior to configuring the nodes
+	err = env.ExistingAddresses.Merge(e.ExistingAddresses)
+	require.NoError(t, err)
+
+	var ocr3Config = keystone.OracleConfigWithSecrets{
+		OracleConfig: keystone.OracleConfig{
+			MaxFaultyOracles: len(wfNodes) / 3,
+		},
+		OCRSecrets: deployment.XXXGenerateTestOCRSecrets(),
+	}
+	var allDons = []keystone.DonCapabilities{wfDon, cwDon, assetDon}
+
+	_, err = kschangeset.ConfigureInitialContractsChangeset(env, kschangeset.InitialContractsCfg{
+		RegistryChainSel: registryChainSel,
+		Dons:             allDons,
+		OCR3Config:       &ocr3Config,
+	})
+	require.NoError(t, err)
+	// TODO: KS-rm_deploy_opt
+	//require.Nil(t, csOut.AddressBook, "no new addresses should be created in configure initial contracts")
+	//require.NoError(t, env.ExistingAddresses.Merge(csOut.AddressBook))
+
+	req := &keystone.GetContractSetsRequest{
+		Chains:      env.Chains,
+		AddressBook: env.ExistingAddresses,
+	}
+
+	contractSetsResp, err := keystone.GetContractSets(lggr, req)
+	require.NoError(t, err)
+	require.Len(t, contractSetsResp.ContractSets, len(env.Chains))
+	// check the registry
+	regChainContracts, ok := contractSetsResp.ContractSets[registryChainSel]
+	require.True(t, ok)
+	gotRegistry := regChainContracts.CapabilitiesRegistry
+	require.NotNil(t, gotRegistry)
+	// validate the registry
+	// check the nodes
+	gotNodes, err := gotRegistry.GetNodes(nil)
+	require.NoError(t, err)
+	require.Len(t, gotNodes, len(allNodes))
+	gotWfNodes, err := gotRegistry.GetNodesByP2PIds(nil, p2pIDs(t, maps.Keys(wfNodes)))
+	require.NoError(t, err)
+	require.Len(t, gotWfNodes, len(wfNodes))
+	gotCwNodes, err := gotRegistry.GetNodesByP2PIds(nil, p2pIDs(t, maps.Keys(cwNodes)))
+	require.NoError(t, err)
+	require.Len(t, gotCwNodes, len(cwNodes))
+	gotAssetNodes, err := gotRegistry.GetNodesByP2PIds(nil, p2pIDs(t, maps.Keys(assetNodes)))
+	require.NoError(t, err)
+	require.Len(t, gotAssetNodes, len(assetNodes))
+
+	// check the dons
+	gotDons, err := gotRegistry.GetDONs(nil)
+	require.NoError(t, err)
+	require.Len(t, gotDons, len(allDons))
+	// todo: add validation of the expected capabilities
+
+	if c.EnableMCMS {
+		// deploy, configure and xfer ownership of MCMS
+		env, err = commonchangeset.ApplyChangesets(t, env, nil, []commonchangeset.ChangesetApplication{
+			{
+				Changeset: commonchangeset.WrapChangeSet(commonchangeset.DeployMCMSWithTimelock),
+				Config: map[uint64]commontypes.MCMSWithTimelockConfig{
+					registryChainSel: {
+						Canceller:         commonchangeset.SingleGroupMCMS(t),
+						Bypasser:          commonchangeset.SingleGroupMCMS(t),
+						Proposer:          commonchangeset.SingleGroupMCMS(t),
+						TimelockExecutors: env.AllDeployerKeys(),
+						TimelockMinDelay:  big.NewInt(0),
+					},
+				},
+			},
+			{
+				Changeset: commonchangeset.WrapChangeSet(kschangeset.TransferAllOwnership),
```

Comment:
I think after this, you need an AcceptOwnership invocation to complete the transfer